### PR TITLE
Milestone: Set up a Hello Microverse project

### PR DESCRIPTION
**Pull request by ahmedeid6842**
In this project, I implemented the following changes:
- Setup linters
- Added a README documentation
- Added `index` and `style.css` file with some styling
- Added `node_modules` and `test.md` in `.gitignore`
- Added MIT Licence

### CODE REVIEW

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ahmedeid6842 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] As required for the project, kindly make sure your **Pull Request title** is more descriptive  in the way tells  what changes are inside. e.g.: **"Milestone: Set up a Hello Microverse project"**

- [x] Also for the project requirements, kindly ensure that your **Pull Request description** includes a summary describing all introduced changes in more detail. For example:
```
In this project, I implemented the following changes:
- Setup linters
- Added a README documenation
- Added `index` and `style.css` file wit some styling
- Added `node_modules` and `test.md` in `.gitignore`
- etc
```

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,222 @@
...
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="https://user-images.githubusercontent.com/57197702/234442761-85f8597b-b552-4c42-8956-90691ac3b638.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
[the remark this answers is not in the document]
Comment (ahmedeid6842) [reply README.md]: Oops, my bad! 😔 I thought I have to add the Microverse logo to all of my projects.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,222 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
```
[the remark this answers is not in the document]
Comment (ahmedeid6842) [reply README.md]: yes, I agree with you but I didn't know that I can remove a section from readme 🙃.

**Comment by ahmedeid6842** [comment]
@tresorsawasawa 
Awesome! That review was incredibly helpful 👏👌. 
I made all the changes you suggested and will definitely keep them in mind for my future projects 😊 ✍️.
leonardo-dicaprio-clapping

**Review by usorfaitheloho — APPROVED** [review]
Hi @ahmedeid6842  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
## Optional suggestions
- As suggested by the previous reviewer, please remove all lines of commented codes from your README file to make your code clean and easy to maintain. Although you did well removing the comments noticed by the previous reviewer 👍 , there are still some commented codes the previous reviewer omitted that should be removed

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**